contracts/axelar-gateway/src/test.rs

```diff
@@ -277,28 +275,32 @@ fn rotate_signers_with_enforce_rotation_delay() {
     let operator = Address::generate(&env);
     let signers = initialize(&env, &client, operator.clone(), 1, 5);
     let new_signers = generate_signers_set(&env, 5, signers.domain_separator.clone());
-    let signers_hash = get_signers_hash(&env, new_signers.signers_set.clone());
-    let data_hash = get_rotation_hash(&env, new_signers.signers_set.clone());
-    let proof = generate_proof(&env, data_hash.clone(), signers);
+    let data_hash = get_rotation_hash(&env, new_signers.signers.clone());
+    let proof = generate_proof(&env, data_hash.clone(), signers.clone());
+    let enforce_rotation_delay = true;
 
     client
         .mock_auths(&[MockAuth {
             address: &operator,
```

Comment:
the caller doesn't matter if rotation delay is `true`. This test case should be for when the delay is being bypassed